demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -547,9 +549,26 @@ def report_summary(self, lint_status: dict):
             print(f"{Colors.Fg.red}{wrapper_fail_pack.fill(fail_pack)}{Colors.reset}")
 
     @staticmethod
-    def _create_report(pkgs_status: dict, path: str):
-        if path:
-            json_path = Path(path) / "lint_report.json"
-            json.dump(fp=json_path.open(mode='w'),
-                      obj=pkgs_status,
-                      indent=4)
+    def _create_failed_unit_tests_report(lint_status: dict, path: str):
+        """
+        Creayes and saves a file containing all failed unit tests
+        :param lint_status: dict
+            Dictionary containing type of failures and corresponding failing tests. Looks like this:
+             lint_status = {
+            "fail_packs_flake8": [],
+            "fail_packs_bandit": [],
+            "fail_packs_mypy": [],
+            "fail_packs_vulture": [],
+            "fail_packs_pylint": [],
+            "fail_packs_pytest": [],
+            "fail_packs_pwsh_analyze": [],
+            "fail_packs_pwsh_test": [],
+            "fail_packs_image": []
+        }
+        :param path: str
+            The path to save the report.
+        """
+        failed_ut = set().union([second_val for val in lint_status.values() for second_val in val])
```

Comment:
```suggestion
        failed_packs = set.union(*lint_status.values())
```
** if changed to sets as mentioned above.
the union is a static method so no need for object init, In addition, I think its better to output failed packages for some reason in this report- so you will be notified, there is no reason for failing if something failed we will be notified.
if not so no need for sets at all only `failed_packs = lint_status[""fail_packs_pytest"]`